tests/integration/README.md

```diff
@@ -118,10 +115,21 @@ In CircleCI, these files can be found in the artifacts section on the test job p
 
 ![CircleCI Artifacts Tab Screenshot](https://circleci.com/docs/assets/img/docs/artifacts.png)
 
+You can also turn on debugging log in your code.
```

Comment:
These can be turned on using --log_output_level flags. Manually setting the logging levels this way should not be encouraged.

Note that CI output level will be set with cimode flag: --istio.test.cimode

Reply:
Done. Add example of how it should look like.